src/main/resources/fabric.mod.json

```diff
@@ -33,10 +33,5 @@
 
   "depends": {
     "fabric-resource-loader-v0": "*"
-  },
-
-  "custom": {
-    "modmenu:api": true,
-    "modmenu:clientsideOnly": true
   }
```

Comment:
You forgot to make this mod a library again:
```suggestion
  "custom" {
    "modmenu": {
      "badges": [ "library" ]
    }
  }
```

Reply:
Actually especially in future versions it shouldn't be marked as a library anymore, because libraries are by default hidden from the list. And the future versions will have many more useful config options and other systems on the malilib config screen.

Reply:
oh, ok